ktor-io/jvm/src/io/ktor/utils/io/ByteBufferChannel.kt

```diff
@@ -1552,9 +1558,11 @@ internal open class ByteBufferChannel(
         }
     }
 
-    override suspend fun write(min: Int, block: (ByteBuffer) -> Unit) {
+    override fun writeAvailable(min: Int, block: (ByteBuffer) -> Unit): Int {
         require(min > 0) { "min should be positive" }
+        require(min <= 4088) { "min should be positive" }
```

Comment:
The error message should reflect the number `4088`.

Reply:
Maybe we need to introduce a constant for this number?

Reply:
Sure, will do